website/content/docs/enterprise/lts.mdx

```diff
@@ -0,0 +1,192 @@
+
+<a id="what" />
+
+## What is long-term support?
+
+**Standard support** for Vault includes:
+
+- performance improvements.
+- feature updates and improvements.
+- bug fixes.
+- security patches.
+- CVE patches to correct vulnerabilities and exposures.
+
+Standard support covers "N&minus;2" versions, which means, at any given time,
+HashiCorp supports the current version ("N") and the two previous versions
+("N&minus;2").
+
+Vault versions typically update 3 times per calendar year, which means that
+standard support for a given Vault version lasts approximately 1 year:
+
+Version | Release date         | Standard support ends
+------- | -------------------- | ----------------------
+1.16    | CY24 Q1 (2024-03-27) | CY25 Q1 (1.19 release) 
+1.15    | CY23 Q3 (2023-09-27) | CY24 Q3 (1.18 release) 
+1.14    | CY23 Q2 (2023-06-21) | CY24 Q2 (1.17 release) 
+1.13    | CY23 Q1 (2023-03-01) | CY24 Q1 (2024-03-27) 
+1.12    | CY22 Q4 (2022-10-12) | CY23 Q3 (2023-09-27) 
+
+**Long-term support** extends support for specific Vault versions for an
+additional year with:
+
+- patches for bugs that may cause outages.
+- patches for critical vulnerabilities and exposures.
+- high-risk security patches.
+
+After the first year, LTS Vault versions move from standard support to long-term
+support for an additional year.
+
+<a id="where" />
+
+## Where do I download the LTS binary?
+
+You do not need to download a separate binary for LTS support. Just make sure to
+select an LTS Vault version (e.g., 1.16, 1.19) when you
+[install](/vault/install) or
+[upgrade](/vault/docs/upgrading) your Vault instance.
+
+<a id="when" />
+
+## When are LTS versions released?
+
+As of Vault Enterprise 1.16, the first release of a calendar year includes
+long-term support.
+
+LTS versions overlap by one year with the previous LTS version entering its
+long-term support window when the new LTS version begins its standard support
+window.
+
+<a id="why" />
+
+## Why should I avoid upgrading from an LTS version to a non-LTS version?
+
+Long-term support is intended for Enterprise customers who cannot upgrade
+regularly to stay within the standard support timeline of one year. The
+goal is to establish a predictable upgrade path with a longer timeline rather
+than extending the lifetime for every Vault version.
+
+Long-term support ensures your Vault version continues to receive critical
+patches for an additional year. If you upgrade to a non-LTS version, you are
+moving your Vault instance to a version that lacks extended support, which means
+your Vault version stops receiving updates once it leaves the standard support
+window:
+
+Version | Expected release | Standard support ends  | Long-term support ends
+------- | ---------------- | ---------------------- | ---------------------
+1.19    | CY25 Q1          | CY26 Q1 (1.22 release) | CY27 Q1 (1.25 release)
+1.18    | CY24 Q3          | CY25 Q3 (1.21 release) | No long-term support
```

Comment:
Say "standard support" instead of "no long-term support" -- again we don't want to push customers to LTS if it's not the right fit for them

Reply:
That's not an appropriate cell value for a column labeled "Long-term support ends". The reality is that version 1.18 and 1.17 *don't* include long term support.